src/pages/home/_Hero.tsx

```diff
@@ -29,13 +29,13 @@ const Hero: FunctionComponent = () => {
         <div className="tw-relative tw-px-sm tw-pb-3xl md:tw-pb-5xl tw-bg-white tw-text-black">
             {[meshLeft, meshRight].map((image, index) => (
                 <div
-                    key={image.src}
+                    key={`mesh-container-${Math.random()}`}
                     className={classNames('tw-hidden lg:tw-block tw-absolute tw-top-0', {
                         ['tw-left-0']: index === 0,
                         ['tw-right-0']: index === 1,
                     })}
                 >
-                    <img src={image.src} alt="" draggable={false} className="w-100 h-800" />
+                    <img src={image.src} alt="" draggable={false} className="tw-w-full tw-max-w-xl" />
```

Comment:
Ah- thank you! The TW max-width wasn't working for me. Do you find that sometimes? Maybe the `tw-full` was what I was missing 🤔 @bretthayes